### DISCUSSION TITLE
Are there any technical resources on SYCL (The compiler architecture view)?

### DISCUSSION THREAD

**Discussion by orion160** [intel/llvm#12233] [Q&A]
I can be on the open source OneAPI compiler or other vendor ;)

**Comment by aelovikov-intel** [comment]
We have these docs in-tree: https://github.com/intel/llvm/tree/sycl/sycl/doc/design, I'm not sure how many of them are up-to-date. Maybe @bader can provide some extra links.

**Reply by bader** [reply]
https://sycl.tech/ has a lot of links to technical resources for SYCL.
Let me tag @illuhad and @keryell to add more specific details about the architecture view of SYCL compilers from other vendors.
